integration/hsm/helpers.go

```diff
@@ -60,9 +61,6 @@ func newTeleportService(t *testing.T, config *servicecfg.Config, name string) *t
 		serviceChannel: make(chan *service.TeleportProcess, 1),
 		errorChannel:   make(chan error, 1),
 	}
-	t.Cleanup(func() {
-		require.NoError(t, s.close(), "error while closing %s during test cleanup", name)
-	})
```

Comment:
curious why this cleanup is removed?

Reply:
just because it doesn't really matter, slows down the test, doesn't usually work if the test failed, and lots of people in https://github.com/gravitational/teleport/issues/14172 were reporting the cleanup errors instead of the actual/initial error